### PR TITLE
Tutorial on using AutoML in sklearn pipeline

### PR DESCRIPTION
**Pull request by bnriiitb**
I have created a tutorial on how to use FLAML's AutoML in the Sklearn pipeline, I feel that this tutorial will help people to leverage AutoML within a pipeline architecture and extensively use it for production-grade solutions.

@sonichi could you please review the PR?

### CODE REVIEW

File: notebook/automl_in_sklearn_pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,963 @@
...
+    "* Allow hyperparameter tuning across the estimators\n",
+    "* Easier to share and collaborate with multiple users (bug fixes, enhancements etc)\n",
+    "* Enforce the implementation and order of steps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### As FLAML's AutoML module can be used a transformer in the Sklearn's pipeline we can get all the benefits of pipeline and there by write extremley clean, and resuable code."
```
Comment (sonichi) [inline notebook/automl_in_sklearn_pipeline.ipynb:70]:
```suggestion
    "#### As FLAML's AutoML module can be used a transformer in the Sklearn's pipeline we can get all the benefits of pipeline and thereby write extremley clean, and resuable code."
```

File: notebook/automl_in_sklearn_pipeline.ipynb [hunk 2]
```diff
@@ -0,0 +1,963 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) 2020-2021 Microsoft Corporation. All rights reserved. \n",
```
Comment (sonichi) [inline notebook/automl_in_sklearn_pipeline.ipynb:7]:
```suggestion
    "Copyright (c) 2021. All rights reserved. \n",
    "Contributed by: @bnriiitb ",
```

**Review by sonichi — COMMENTED** [review]
Looks good to me. I have some minor comments.

**Comment by bnriiitb** [comment]
> Looks good to me. I have some minor comments.

incorporated the suggestions. kindly review.

**Review by sonichi — APPROVED** [review]
close #20

**Pull request merged**